### PR TITLE
Couple shell membrane equations to TFMP equations

### PR DESCRIPTION
**Pull request by acochrane**
Enumerate controls the kind of shell moment tensor used

### CODE REVIEW

**Comment by wortiz** [comment]
I didn't see anything that stuck out as being wrong, but there is a lot here.

I asked @tjiptowi if he could take a look at it as well.

Did you want to add any test problems for this, or are they covered under your tfmp tests already?

File: src/mm_fill_shell.c [hunk 1]
```diff
@@ -16285,15 +16496,18 @@ assemble_shell_tfmp(double time,   /* Time */
...
 	  adv = 0.0;
 	  adv += phi_i*clip_strength*(1.0-S)*(1.0-S)*dh_dtime;
 	}
       }
       /* Assemble diffusion term */
       diff = 0.0;
+   
       if( T_DIFFUSION ) {
-	//diff += phi_i;
+
```
Comment (wortiz) [inline src/mm_fill_shell.c]: Similar to advection before should the diffusion related parts be removed here

File: src/mm_fill_shell.c [hunk 2]
```diff
@@ -16377,19 +16594,21 @@ assemble_shell_tfmp(double time,   /* Time */
...
 	    mass *= etm_mass_eqn;
 	  }
+
+	  
 	  // Assemble advection term
 	  adv = 0.0;
 
 	  if ( T_ADVECTION ) {
-	    adv += phi_i*phi_j*dh_dtime;
+	    adv += 0.0;
```
Comment (wortiz) [inline src/mm_fill_shell.c]: advection again

File: src/mm_fill_terms.c [hunk 3]
```diff
@@ -12335,50 +12335,103 @@ if ( pd->v[LIGHT_INTM] )
       siz = sizeof(double)*DIM*DIM*MDE;
       memset(&(fv->d_grad_poynt_dmesh[1][0][0][0]),0, siz);
       for ( i=0; i<vdofs; i++)
+  {
+    T_i = *esp->poynt[1][i];
```
Comment (wortiz) [inline src/mm_fill_terms.c]: Looks like somehow indentation for this and  LIGHT_INTD got messed up

**Comment by acochrane** [comment]
Let me validate that this still works before squash & merge.

**Comment by wortiz** [comment]
Go ahead and merge when ready.

**Pull request merged**